src/modules/uavcan/actuators/esc.cpp

```diff
@@ -49,12 +49,24 @@ UavcanEscController::UavcanEscController(uavcan::INode &node) :
 	_uavcan_sub_status(node),
 	_orb_timer(node)
 {
+	if (_perfcnt_invalid_input == nullptr) {
+		errx(1, "uavcan: couldn't allocate _perfcnt_invalid_input");
+	}
+
+	if (_perfcnt_scaling_error == nullptr) {
+		errx(1, "uavcan: couldn't allocate _perfcnt_scaling_error");
+	}
+
+	if (_perfcnt_broadcast_elapsed == nullptr) {
+		errx(1, "uavcan: couldn't allocate _perfcnt_broadcast_elapsed");
+	}
 }
 
 UavcanEscController::~UavcanEscController()
 {
 	perf_free(_perfcnt_invalid_input);
 	perf_free(_perfcnt_scaling_error);
+	perf_free(_perfcnt_broadcast_elapsed);
```

Comment:
Sure, most of them can be removed. Why their number is a concern? According to my profiling results, their performance impact is negligible.